docs/configuration/agent/configuration.md

```diff
@@ -241,6 +241,21 @@ The recommended approach for production deployments is to use ConfigMap entries
 - **Default**: `""` (empty)
 - **Example**: `"redis-password"`
 
+### Resource Proxy Configuration
+
+#### Enable Resource Proxy
+- **Command Line Flag**: `--enable-resource-proxy`
+- **Environment Variable**: `ARGOCD_AGENT_ENABLE_RESOURCE_PROXY`
+- **ConfigMap Entry**: *(Not available in ConfigMap)*
```

Comment:
Nit: Should the configmap entry be `agent.resource-proxy.enable`?

Reply:
Good catch, fixed.